shipyard-controller/docs/swagger.yaml

```diff
@@ -533,7 +514,7 @@ info:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   title: Control Plane API
-  version: "1.0"
+  version: develop
```

Comment:
same question here

Reply:
yes, it gets replaced here https://github.com/keptn/keptn/blob/12c9f275ad71c16923b9cc7eed700ec05c1ecb49/shipyard-controller/Dockerfile#L40